remix-lib/src/universalDapp.js

```diff
@@ -333,7 +335,7 @@ module.exports = class UniversalDApp {
 
           if (err) return next(err)
           if (!address) return next('No accounts available')
-          if (executionContext.isVM() && !self.accounts[address]) {
+          if (self.executionContext.isVM() && !self.accounts[address]) {
```

Comment:
Can you use arrow funtions in the above code to use `this` instead of `self` (in another PR if needed :) )